q2_diversity/__init__.py

```diff
@@ -6,8 +6,8 @@
 # The full license is in the file LICENSE, distributed with this software.
 # ----------------------------------------------------------------------------
 
-__version__ = "0.0.0-dev"
-
 from ._beta import beta_diversity
 
+__version__ = "0.0.0-dev"
```

Comment:
This hasn't changed (just moved location in this file). Note this is what we're doing in the other plugins right now:
https://github.com/qiime2/q2-feature-table/blob/master/q2_feature_table/__init__.py#L15
https://github.com/qiime2/q2-types/blob/master/q2_types/__init__.py#L12

Mind if we just leave it that way for the moment, and change it in all the repos at the same time?